tests/network/vmi_networking.go

```diff
@@ -302,15 +302,6 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			}, 15)
 			Expect(err).ToNot(HaveOccurred())
 		})
-
-		It("[test_id:1775]should start a VMI with no network", func() {
```

Comment:
With this change, you should also remove the redundant context. (I would use its text to describe the test instead of what exists now in the `It` one).